addons/website_event_sale/tests/common.py

```diff
@@ -34,28 +39,72 @@ def setUpClass(cls):
         }])
         cls.currency_test = cls.env['res.currency'].create({
             'name': 'eventX',
-            'rate': 10,
             'rounding': 0.01,
             'symbol': 'EX',
         })
         cls.partner = cls.env['res.partner'].create({'name': 'test'})
-        cls.new_company = cls.env['res.company'].create({
-            'currency_id': cls.env.ref('base.EUR').id,
-            'name': 'Great Company EUR',
-            'partner_id': cls.partner.id,
-        })
```

Comment:
Could you please explain 
* If we need to change the company used, why ?
* Or, if it just works to simplify things,
  * If we had to use another company before and why we don't need it now?
  * Or if it was just too complex and never required to do so

Then, depending on the case, add it to the commit message?

Reply:
I just mention it in the new commit message. It was because I had some difficulties to have the right rate. But thanks to your help it is not an issue anymore.